src/main/java/standardLibrary/stdlib.fg

```diff
@@ -0,0 +1,32 @@
+/*
```

Comment:
This file should not be in `src/main/java` it should probably be in `src/main/flowg`